src/main/java/org/dependencytrack/persistence/ComponentQueryManager.java

```diff
@@ -135,7 +135,7 @@ public List<Component> getAllComponents() {
     @SuppressWarnings("unchecked")
     public List<Component> getAllComponents(Project project) {
         final Query<Component> query = pm.newQuery(Component.class, "project == :project");
-        query.getFetchPlan().setMaxFetchDepth(2);
+        query.getFetchPlan().setMaxFetchDepth(3);
```

Comment:
What data are we trying to capture with this increase in fetch depth? I wonder if [`FetchGroup`](https://www.datanucleus.org/products/accessplatform_6_0/jdo/persistence.html#fetch_groups)s would be a better tool to avoid over-fetching of data that is not strictly needed.

Reply:
The frontend PR contains code for rendering crypto properties. They form an additional layer in the persistence model which must be fetched for rendering (fetch depth 2 was not enough). To keep things simple I just raised the fetch depth to make this work. We could define a dedicated crypto component fetch group and use this only when crypto assets are queried.

Reply:
OK, I'd propose we deal with this once everything else in place, as it's just an optimization and DataNucleus' fetch behavior can be quite fiddly to tune.